README.md

```diff
@@ -505,7 +505,7 @@ interface {
 
 ### Endpoint level commands
 
-#### [Invoke a Command Classes API method]
```

Comment:
this was meant to link to https://zwave-js.github.io/node-zwave-js/#/api/endpoint?id=invokeccapi but I forgot to add the link. Can we revert and add the link in ()?

Reply:
Sure, I'll fix them.